Wikipedia/Localizations/qqq.lproj/Localizable.strings

```diff
@@ -59,6 +59,9 @@
 "account-creation-captcha-required" = "Alert message shown when account creation require a captcha verification";
 "account-creation-captcha-retry" = "Alert message shown when the captcha the user entered did not match the captcha image";
 "account-creation-captcha-obtaining" = "Alert shown when user wants a new captcha when creating account";
+"account-creation-captcha-title" = "Title for account creation CAPTCHA interface";
```

Comment:
Any time a new translation string is added this is the place you must add a description of the string. This is presented to translators on TWN to provide context.